spec/spec_helper.rb

```diff
@@ -1,28 +1,24 @@
 ENV["RAILS_ENV"] ||= 'test'
 require "bundler/setup"
 
-def coverage_needed?
-  ENV['COVERAGE'] || ENV['TRAVIS']
-end
-
 def ci_build?
-  ENV['TRAVIS'] || ENV['CIRCLE']
+  ENV['CI']
 end
 
-if coverage_needed?
-  require 'simplecov'
-  require 'coveralls'
-  SimpleCov.root(File.expand_path('../..', __FILE__))
-  SimpleCov.formatter = Coveralls::SimpleCov::Formatter
-  SimpleCov.start('rails') do
-    add_filter '/.internal_test_app'
-    add_filter '/lib/generators'
-    add_filter '/spec'
-    add_filter '/tasks'
-    add_filter '/lib/hyrax/version.rb'
-    add_filter '/lib/hyrax/engine.rb'
-  end
-  SimpleCov.command_name 'spec'
+require 'simplecov'
```

Comment:
This just duplicates what is being introduced in https://github.com/samvera/hyrax/pull/3810, I can also remove this.